
# List of papers for PhD candidacy exam


Tensor decomposition:

* Matrix factorization techniques for recommender systems ([link](http://ieeexplore.ieee.org/document/5197422/))

* Temporal Collaborative Filtering with Bayesian Probabilistic Tensor Factorization ([link](https://www.cs.cmu.edu/~jgc/publication/PublicationPDF/Temporal_Collaborative_Filtering_With_Bayesian_Probabilidtic_Tensor_Factorization.pdf))

* Tensor decomposition for multiple-tissue gene expression experiments ([link](http://www.nature.com/ng/journal/v48/n9/full/ng.3624.html))


Co-expression network analysis:

* A latent factor model with a mixture of sparse and dense factors to model gene expression data with confounding effects ([link](https://arxiv.org/abs/1310.4792))

* Bayesian group latent factor analysis with structured sparsity ([link](http://jmlr.org/papers/volume17/14-472/14-472.pdf))

* Context-specific differential gene co-expression networks via Bayesian biclustering models ([link](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004791))

* Sharing and specificity of co-expression networks across 35 human tissues ([link](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004220))

* Co-expression networks reveal the tissue-specific regulation of transcription and splicing ([link](http://biorxiv.org/content/early/2016/10/02/078741))


eQTLs:

* Learning a Prior on Regulatory Potential from eQTL Data ([link](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000358))

* Local genetic effects on gene expression across 44 human tissues ([link](http://biorxiv.org/content/early/2016/09/09/074450))

* Distant regulatory effects of genetic variation in multiple human tissues ([link](http://biorxiv.org/content/early/2016/09/09/074419))


GTEx flagship:

* The Genotype-Tissue Expression (GTEx) project ([link](http://www.nature.com/ng/journal/v45/n6/full/ng.2653.html))

* The human transcriptome across tissues and individuals ([link](http://science.sciencemag.org/content/348/6235/660))

* The Genotype-Tissue Expression (GTEx) pilot analysis: Multitissue gene regulation in humans ([link](http://science.sciencemag.org/content/348/6235/648))


Cell type heterogeneity correction in association study:

* Epigenome-wide association studies without the need for cell-type composition ([link](http://www.nature.com/nmeth/journal/v11/n3/full/nmeth.2815.html))

* Sparse PCA corrects for cell type heterogeneity in epigenome-wide association studies ([link](Sparse PCA corrects for cell type heterogeneity in epigenome-wide association studies))

* Correcting for cell-type heterogeneity in DNA methylation: a comprehensive evaluation ([link](http://www.nature.com/nmeth/journal/v14/n3/full/nmeth.4190.html?WT.feed_name=subjects_biological-sciences))


Joint modeling of genetics and X (X=brain imaging)

* Probabilistic Modeling of Imaging, Genetics and Diagnosis ([link](http://ieeexplore.ieee.org/document/7404010/))

* Generative Method to Discover Genetically Driven Image Biomarkers ([link](https://link.springer.com/chapter/10.1007/978-3-319-19992-4_3))

* Spherical Topic Models for Imaging Phenotype Discovery in Genetic Studies ([link](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4337963/))

* Joint modeling of imaging and genetics ([link](https://www.ncbi.nlm.nih.gov/pubmed/24684016))


Others:

* A model-based approach for analysis of spatial structure in genetic data ([link](http://www.nature.com/ng/journal/v44/n6/abs/ng.2285.html))

* A gene-based association method for mapping traits using reference transcriptome data ([link](http://www.nature.com/ng/journal/v47/n9/full/ng.3367.html))

* A Unified Statistical Framework for Single Cell and Bulk RNA Sequencing Data ([link](https://arxiv.org/abs/1609.08028))

* Deep learning for computational biology ([link](http://msb.embopress.org/content/12/7/878))

* Scaling probabilistic models of genetic variation to millions of humans ([link](http://www.nature.com/ng/journal/v48/n12/full/ng.3710.html))

* Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles ([link](http://www.pnas.org/content/102/43/15545.abstract))




many more to come..
