
# A probabilistic perspective on tissue-specific co-expression networks: genetic regulation and biological consequences


## 1. Genotype-Tissue Expression (GTEx) project -- tissues on top of cell types:
1. **The Genotype-Tissue Expression (GTEx) project** ([link](http://www.nature.com/ng/journal/v45/n6/full/ng.2653.html)). Lonsdale, J., Thomas, J., Salvatore, M., Phillips, R., Lo, E., Shad, S., ... Moore, H. F. (2013, 06). *Nature Genetics*.
1. **The human transcriptome across tissues and individuals** ([link](http://science.sciencemag.org/content/348/6235/660)). Melé, M., Ferreira, P. G., Reverter, F., DeLuca, D. S., Monlong, J., Sammeth, M., ... Guigó, R. (2015). *Science*.
1. **The Genotype-Tissue Expression (GTEx) pilot analysis: Multitissue gene regulation in humans** ([link](http://science.sciencemag.org/content/348/6235/648)). Ardlie, K. G., Deluca, D. S., Segrè, A. V., Sullivan, T. J., Young, T. R., Gelfand, E. T., ... Dermitzakis, E. T. (2015). *Science*.
1. **A Unified Statistical Framework for Single Cell and Bulk RNA Sequencing Data** ([link](https://arxiv.org/abs/1609.08028)). Zhu, L., Lei, J., Devlin, B., & Roeder, K. (2016, 09). *ArXiv e-prints*.

## 2. Gene co-expression network:
1. **<u style="background-color:rgb(0,255,0)">A latent factor model with a mixture of sparse and dense factors to model gene expression data with confounding effects</u>** ([link](https://arxiv.org/abs/1310.4792)). Gao, C., Brown, C. D., & Engelhardt, B. E. (2013, 10). *ArXiv e-prints*.
1. **<u>Context-specific differential gene co-expression networks via Bayesian biclustering models</u>** ([link](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004791)). Gao, C., McDowell, I. C., Zhao, S., Brown, C. D., & Engelhardt, B. E. (2016, 07). *PLOS Computational Biology*.
1. **<u>Bayesian group latent factor analysis with structured sparsity</u>** ([link](http://jmlr.org/papers/volume17/14-472/14-472.pdf)). Zhao, S., Gao, C., Mukherjee, S., & Engelhardt, B. E. (2016). *Journal of Machine Learning Research (JMLR)*.
1. **Sharing and specificity of co-expression networks across 35 human tissues** ([link](http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1004220)). Pierson, E., the GTEx Consortium, Koller, D., Battle, A., & Mostafavi, S. (2015, 05). *PLOS Computational Biology*.
1. **Co-expression networks reveal the tissue-specific regulation of transcription and splicing** ([link](http://biorxiv.org/content/early/2016/10/02/078741)). Saha, A., Kim, Y., Gewirtz, A. D. H., Jo, B., Gao, C., McDowell, I. C., ... Battle, A. (2016). *bioRxiv*.
1. **Understanding multicellular function and disease with human tissue-specific networks** ([link](https://www.nature.com/ng/journal/v47/n6/full/ng.3259.html)). Greene, C. S., Krishnan, A., Wong, A. K., Ricciotti, E., Zelaya, R. A., Himmelstein, D. S., ... Troyanskaya, O. G. (2015, 06). *Nature Genetics*.

## 3. Context specificity -- a tensor view:
1. **Temporal Collaborative Filtering with Bayesian Probabilistic Tensor Factorization** ([link](https://www.cs.cmu.edu/~jgc/publication/PublicationPDF/Temporal_Collaborative_Filtering_With_Bayesian_Probabilidtic_Tensor_Factorization.pdf)). Xiong, L., Chen, X., Huang, T. K., Schneider, J., & Carbonell, J. G. (2010). *Siam data mining 2010 (sdm 10)*.
1. **Tensor decomposition for multiple-tissue gene expression experiments** ([link](http://www.nature.com/ng/journal/v48/n9/full/ng.3624.html)). Hore, V., Vinuela, A., Buil, A., Knight, J., McCarthy, M. I., Small, K., & Marchini, J. (2016, 09). *Nature Genetics*.

## 4. Epigenomics and heterogeneity:
1. **An integrated encyclopedia of DNA elements in the human genome** ([link](https://www.nature.com/nature/journal/v489/n7414/full/nature11247.html)). Consortium, T. E. P. (2012, 09). *Nature*.
1. **Integrative analysis of 111 reference human epigenomes**. ([link](http://www.nature.com/nature/journal/v518/n7539/full/nature14248.html)). Consortium, R. E., Kundaje, A., Meuleman, W., Ernst, J., Bilenky, M., Yen, A., ... Kellis, M. (2015, 02). *Nature*.
1. **ChromHMM: automating chromatin-state discovery and characterization** ([link](http://www.nature.com/nmeth/journal/v9/n3/full/nmeth.1906.html)). Ernst, J., & Kellis, M. (2012, 03). *Nature Methods*.
1. **Epigenome-wide association studies without the need for cell-type composition** ([link](http://www.nature.com/nmeth/journal/v11/n3/full/nmeth.2815.html)). Zou, J., Lippert, C., Heckerman, D., Aryee, M., & Listgarten, J. (2014, 03). *Nature Methods*.
1. **Sparse PCA corrects for cell type heterogeneity in epigenome-wide association studies** ([link](http://www.nature.com/nmeth/journal/v13/n5/full/nmeth.3809.html)). Rahmani, E., Zaitlen, N., Baran, Y., Eng, C., Hu, D., Galanter, J., ... Halperin, E. (2016, 05). *Nature Methods*.

## 5. Genetics of gene expression -- expression quantitative trait loci (eQTLs):
1. **Local genetic effects on gene expression across 44 human tissues** ([link](http://biorxiv.org/content/early/2016/09/09/074450)). Aguet, F., Brown, A. A., Castel, S., Davis, J. R., Mohammadi, P., Segre, A. V., ... Montgomery, S. B. (2016). *bioRxiv*.
1. **Distant regulatory effects of genetic variation in multiple human tissues** ([link](http://biorxiv.org/content/early/2016/09/09/074419)). Jo, B., He, Y., Strober, B. J., Parsana, P., Aguet, F., Brown, A. A., ... Battle, A. (2016). *bioRxiv*.
1. **The impact of rare variation on gene expression across tissues** ([link](http://biorxiv.org/content/early/2016/09/09/074443)). Li, X., Kim, Y., Tsang, E. K., Davis, J. R., Damani, F. N., Chiang, C., ... Montgomery, S. B. (2016). *bioRxiv*.
1. **<u>Learning a Prior on Regulatory Potential from eQTL Data</u>** ([link](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1000358)). Lee, S.-I., Dudley, A. M., Drubin, D., Silver, P. A., Krogan, N. J., Pe’er, D., & Koller, D. (2009, 01). *PLOS Genetics*.
1. **<u>Adaptive Multi-Task Lasso: with Application to eQTL Detection</u>** ([link](http://www.cs.cmu.edu/~seunghak/NIPS2010_0499.pdf)). Lee, S., Zhu, J., & Xing, E. P. (2010). *Advances in neural information processing systems (NIPS)*.
1. **<u>A statistical framework for joint eqtl analysis in multiple tissues</u>** ([link](http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1003486)). Flutre, T., Wen, X., Pritchard, J., & Stephens, M. (2013, 05). *PLOS Genetics*.
1. **An Efficient Nonlinear Regression Approach for Genome-wide Detection of Marginal and Interacting Genetic Variations** ([link](http://online.liebertpub.com/doi/10.1089/cmb.2015.0202)). Lee, S., Lozano, A. C., Kambadur, P., & Xing, E. P. (2016). *Journal of Computational Biology*.
1. **A gene-based association method for mapping traits using reference transcriptome data** ([link](http://www.nature.com/ng/journal/v47/n9/full/ng.3367.html)). Gamazon, E. R., Wheeler, H. E., Shah, K. P., Mozaffari, S. V., Aquino-Michaels, K., Carroll, R. J., ... Im, H. K. (2015, 09). *Nature Genetics*.

## 6. Joint modeling of genetics and X (X=bio-imaging):
1. **Probabilistic Modeling of Imaging, Genetics and Diagnosis** ([link](http://ieeexplore.ieee.org/document/7404010/)). Batmanghelich, N. K., Dalca, A., Quon, G., Sabuncu, M., & Golland, P. (2016, 07). *IEEE Transactions on Medical Imaging*.
1. **Generative Method to Discover Genetically Driven Image Biomarkers** ([link](https://link.springer.com/chapter/10.1007/978-3-319-19992-4_3)). Batmanghelich, N. K., Saeedi, A., Cho, M., Estepar, R. S. J., & Golland, P. (2015). *Information processing in medical imaging*.


**<u>title</u>**: special treatment



[[back](http://shuo-yang.com)]
